content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -220,7 +220,14 @@ of the controller that should implement the class.
 {{< codenew file="service/networking/external-lb.yaml" >}}
 
 IngressClass resources contain an optional parameters field. This can be used to
-reference additional configuration for this class.
+reference additional implementation-specific configuration for this class.
+
+#### Namespace-scoped parameters
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
```

Comment:
We also need to document the feature gate here: https://github.com/kubernetes/website/blob/master/content/en/docs/reference/command-line-tools-reference/feature-gates.md. I'm not sure what the standard is for this, but we may also need to reference that feature gate in this file.